tests/show_bgp_neighbor_test.py

```diff
@@ -0,0 +1,125 @@
+import importlib
+import os
+
+import pytest
+
+from click.testing import CliRunner
+from .bgp_commands_input.bgp_neighbor_test_vector import *
+
+
+def executor(test_vector, show):
+    runner = CliRunner()
+    input = testData[test_vector]
+    if test_vector.startswith('bgp_v6'):
+        exec_cmd = show.cli.commands["ipv6"].commands["bgp"].commands["neighbors"]
+    else:
+        exec_cmd = show.cli.commands["ip"].commands["bgp"].commands["neighbors"]
+
+    result = runner.invoke(exec_cmd, input['args'])
+
+    print(result.exit_code)
+    print(result.output)
+
+    if input['rc'] == 0:
+        assert result.exit_code == 0
+    else:
+        assert result.exit_code == input['rc']
+
+    if 'rc_err_msg' in input:
+        output = result.output.strip().split("\n")[-1]
+        assert input['rc_err_msg'] == output
+
+    if 'rc_output' in input:
+        assert result.output == input['rc_output']
+
+    if 'rc_warning_msg' in input:
+        output = result.output.strip().split("\n")[0]
+        assert input['rc_warning_msg'] in output
+
+
+class TestBgpNeighbors(object):
+
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        from .mock_tables import mock_single_asic
+        importlib.reload(mock_single_asic)
+        from .mock_tables import dbconnector
+        dbconnector.load_namespace_config()
+
+    @pytest.mark.parametrize('setup_single_bgp_instance, test_vector',
+                             [
+                                ('bgp_v4_neighbors_output', 'bgp_v4_neighbors'),
+                                ('bgp_v4_neighbors_output', 'bgp_v4_neighbors'),
```

Comment:
problem with parameterizing test cases is it can be confusing to find out which one failed if the variables have same names.